src/components/memorygame/MemoryGameHowToPlayPopover/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { QuestionCircleTwoTone } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import { Popover } from "antd";
+import "./index.css";
+
+const HowToPlayPopover = () => {
```

Comment:
this component is named wrong because Popover refers to the thing that pops out but this is a component for the icon itself

so it should be called HowToPlayIcon

Reply:
if it were a popover i'd expect it takes a single prop `children` and renders them
```
<HowToPlayPopover>
my stuff that gets wrapped by a popover
</
```

^ expectations for a component called popover

Reply:
I will name it as `QuestionMarkIcon` .So, co-developers will now what it is even without trying popover functionality.

Reply:
git mv used to change name. PTAL